contentcuration/contentcuration/templates/settings/issues.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE HTML>
+{% extends "settings/settings.html" %}
+{% load staticfiles %}
+{% load translation_tags %}
+{% load license_tags %}
+{% load i18n %}
+
+{% block settingscontent %}
+<div id="issue_report">
+	<h4>{% trans "About the Kolibri Studio Beta" %}</h4>
+	<p>{% blocktrans %}Kolibri Studio is Beta software, which means it is currently undergoing active development. As such, it is being regularly updated, and some changes could cause unexpected behavior or challenges (also known as "issues"). While we make every effort to ensure your work is saved and backed up, <b>during active development it's possible that some issues could cause data loss</b>, so we recommend you save and publish changes regularly, and notify us of issues as soon as they occur to help us resolve them. (See below for instructions on how to report issues.){% endblocktrans %}</p>
+	<br/>
+	<p>{% trans "We appreciate your patience and assistance as we work to improve Kolibri Studio and make it an even better tool for importing and curating educational content for Kolibri! " %}</p>
+	<hr/>
+	<div class="container-fluid">
+		<h5>{% trans "Notable Outstanding Issues (Updated: Oct 29, 2018)" %}</h5>
```

Comment:
Is it possible to make Updated a separate translation block, and generate a localized version of the date somehow? This way "Notable Outstanding Issues" and "Updated" won't need updated translation strings when the date changes.

Reply:
Updated!